.github/workflows/cert-renewal.yaml

```diff
@@ -0,0 +1,140 @@
+name: cert-renewal
+on:
+  workflow_call:
+    inputs:
+      CERT_CONTACT_EMAIL:
+        description: 'Email will be sent when cert almost expired'
+        type: string
+        required: true
+      CERT_STAGING:
+        description: 'True or False'
+        type: string
+        required: true
+        default: 'True'
+      ROUTE53_ZONE_NAME:
+        description: 'Route53 Hosted Zone name'
+        type: string
+        required: true
+      ROUTE53_RECORD_NAME:
+        description: 'Route53 record name'
+        type: string
+        required: true
+      MANUAL_LETSENCRYPT_REPO:
+        description: 'manual-letsencrypt repo useful to work with dev fork'
+        type: string
+        default: 'https://github.com/datajoint/manual-letsencrypt.git'
+      MANUAL_LETSENCRYPT_BRANCH:
+        description: 'manual-letsencrypt repo useful to work with dev branch'
+        type: string
+        default: 'master'
+      CALLER_REPO_SUBPATH_FOR_CERT:
+        description: 'a subpath in the caller repo to store the cert'
+        type: string
+      PR_REVIEWERS:
+        description: 'comma separated list of github usernames'
+        type: string
+    secrets:
+      ROUTE53_ZONE_ID:
+        description: 'Route53 Hosted Zone ID'
+        required: true
+      INSTANCE_VPC_SECURITY_GROUP_IDS:
+        description: 'list of sg ids in base64 encoded'
+        required: true
+      INSTANCE_SUBNET_ID:
+        required: true
+      AWS_REGION:
+        required: true
+      AWS_ACCESS_KEY_ID:
+        required: true
+      AWS_SECRET_ACCESS_KEY:
+        required: true
+      SSH_PRIVATE_KEY:
+        description: 'ssh private key to access the instance in base64 encoded'
+        required: true
+
+jobs:
+  cert-renewal:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout caller repo
+        uses: actions/checkout@v4
+        with:
+          path: caller-repo
+      - name: Checkout manual-letsencrypt
+        uses: actions/checkout@v4
+        with:
+          repository: yambottle/manual-letsencrypt
```

Comment:
Reminder: change to `datajoint/manual-letsencrypt`

Reply:
Actually, you could probably use ${{inputs.MANUAL_LETSENCRYPT_REPO}} since you've parametrized this value elsewhere.

Reply:
`${{inputs.MANUAL_LETSENCRYPT_REPO}}` is a git HTTPS url, which surprisingly won't be accepted by actions/checkout... just to make it simple, so I hardcoded here

Reply:
Yeah, you are correct, let me fix that